README.md

```diff
@@ -128,7 +132,7 @@ public class TenjinExampleScript : MonoBehaviour {
       instance.Connect();
 
       // Sets SKAdNetwork Conversion Value
-      instance.UpdateConversionValue(1);
+      instance.UpdateConversionValue(“your 6 bit value”);
```

Comment:
`<YOUR 6 bit value>` might be more consistent with https://github.com/tenjin/tenjin-ios-sdk/pull/67 and also could indicate to the user that this value needs to be changed

Reply:
Thank you, updated